<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>event_manager_window</class>
 <widget class="QMainWindow" name="event_manager_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>your library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>new event</string>
     </attribute>
     <widget class="QLineEdit" name="theme_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
      <property name="placeholderText">
       <string>event theme</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>event date</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="sp_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>511</width>
        <height>171</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>240</number>
      </attribute>
      <column>
       <property name="text">
        <string>speaker name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="b_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>511</width>
        <height>171</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>240</number>
      </attribute>
      <column>
       <property name="text">
        <string>book name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="new_event_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>450</y>
        <width>251</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>create event</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>new speaker</string>
     </attribute>
     <widget class="QLineEdit" name="phone_edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="placeholderText">
       <string>phone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_speaker_button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>create speaker account</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
